examples/fabric-admin/commands/common/StayActiveSender.h

```diff
@@ -0,0 +1,78 @@
+/*
+ *
+ *    Copyright (c) 2024 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#include <stddef.h>
+
+#include <app/InteractionModelEngine.h>
+#include <lib/core/ScopedNodeId.h>
+#include <messaging/ExchangeMgr.h>
+
+/**
+ * @brief StayActiveSender contains all the data and methods needed for active period extension of an ICD client.
+ */
```

Comment:
Please update this with lifetime description and usage: this class makes assumptions on when it auto-deletes, it looks like it auto-deletes if EstablishSessionToPeer returns CHIP_NO_ERROR but is expected to be deleted by the owner if no establish is called or if establish returns an error.

Reply:
Added comment, but also lifetime is now managed in StayActiveSender